README.md

```diff
@@ -26,8 +25,12 @@ we clarify here for external readers:
 
 ## License
 
-The documents in this project are licensed under the
-[CC-By 3.0 License](LICENSE), which encourages you to share these documents. See
-<https://creativecommons.org/licenses/by/3.0/> for more details.
+The documents in this project are based on the [Google Engineering Practices][1]
+documentation. They are licensed under the [CC-By 3.0 License](LICENSE), which
+encourages you to share these documents.
```

Comment:
> I can't add a comment to the License sections, but do you think as per the CC-By 3.0 terms we should maybe attribute the original work?

@oestoer: Added this. Do you think this should be sufficient?

Reply:
Thanks, I think it's good